src/preset.ts

```diff
@@ -392,28 +504,42 @@ function getVendorForConfigurePresetHelper(folder: string, preset: ConfigurePres
   return preset.vendor || null;
 }
 
-export async function expandConfigurePreset(folder: string,
-                                            name: string,
-                                            workspaceFolder: string,
-                                            sourceDir: string,
-                                            allowUserPreset: boolean = false): Promise<ConfigurePreset | null> {
-  const refs = referencedConfigurePresets.get(folder);
-  if (!refs) {
-    referencedConfigurePresets.set(folder, new Set());
+async function getHostSystemName(): Promise<string> {
+  if (platform() === "win32") {
+    return "Windows";
   } else {
-    refs.clear();
+    return promisify(exec)('uname -s').then(
```

Comment:
Can you use `proc.execute` instead?  There should be examples in the code of how to use it.